test/e2e/case5_policy_automation_test.go

```diff
@@ -859,124 +861,249 @@ var _ = Describe("Test policy automation", Ordered, func() {
 
 				return len(ansiblejobList.Items)
 			}, 30, 1).Should(Equal(0))
+
+			cleanupPolicyAutomation()
 		})
-		It("Test manual run", func() {
-			By("Creating an policyAutomation with mode=disable")
-			_, err := utils.KubectlWithOutput("apply",
-				"-f", "../resources/case5_policy_automation/case5-policy-automation-disable.yaml",
-				"-n", testNamespace)
-			Expect(err).ShouldNot(HaveOccurred())
-			By("Applying manual run annotation")
-			_, err = utils.KubectlWithOutput(
-				"annotate",
-				"policyautomation",
-				"-n",
-				testNamespace,
-				automationName,
-				"--overwrite",
-				"policy.open-cluster-management.io/rerun=true",
-			)
-			Expect(err).ShouldNot(HaveOccurred())
-			By("Should only create one ansiblejob which include 3 noncompliant target_clusters")
-			Eventually(func() interface{} {
-				ansiblejobList, err := clientHubDynamic.Resource(gvrAnsibleJob).Namespace(testNamespace).List(
-					context.TODO(), metav1.ListOptions{},
-				)
-				Expect(err).ToNot(HaveOccurred())
-				_, err = utils.KubectlWithOutput("get", "ansiblejobs", "-n", testNamespace)
-				Expect(err).ShouldNot(HaveOccurred())
+	})
 
-				return len(ansiblejobList.Items)
-			}, 30, 1).Should(Equal(1))
+	Describe("Test PolicyAutomation Maunul run", func() {
+		Describe("Test manual run", func() {
+			It("should no issue when init policyAutomation with disable and manual ", func() {
```

Comment:
I just copied and pasted Test manual run and added this part inside `describe` for easy partial testing